### PR TITLE
Slightly un-nerfs the cmo(as a treat(medical belt edition))

### PR DESCRIPTION
**Pull request by Draggeru**
## About The Pull Request

Gives the cmo their own medical belt that is normal sized instead of bulky.

## Why It's Good For The Game

When all of the belts in the game were rebalanced it was treated like all belts are equal when in fact not all belts are equal. The CMO spawns with a compact defib in their locker, which is still normal-sized. This gives the CMO the ability to be able to use their defib without playing a frustrating inventory management game of: take their medical belt off, get their defib out, put their tools on the floor and put their defib on, use their defib to revive someone away from medbay(or in medbay), take their defib off to put back in their bag, and finally replace their medical belt back on their waist. 

This PR comes from a lovely sprite by Cimika, who gave me permission to use it for this pr.

## Changelog
:cl:
add: Gives the cmo a neat and special belt that's normal-sized, making field medic cmo gameplay more viable once again.
:cl:

### CODE REVIEW

**Comment by Draggeru** [comment]
Melbert said the add tag is more fitting, adjusted the changelog

**Comment by Farquaar** [comment]
Instead of saying the belt is "smaller", maybe say it's more "space-efficient". That way, you don't give the impression that it fits less.

**Comment by Draggeru** [comment]
After talking in coder bus there is an alternative solution. I wanna hear what maintainers think first though. Instead of giving the cmo a smaller belt let compact defibs fit on the suit storage slot of all medical exosuits. A-And giving the cmo their own personal reskinned belt thats bulky because damnit I want to look like a special snowflake.

**Comment by Shadowflame909** [comment]
That nerf should just be reverted instead.

Nukies Chest-Rigs got blammered

**Comment by GuillaumePrata** [comment]
The CMO having their own unique belt like the CE would be nice but this everlasting trend of Command players trying to dodge balance changes by getting improved gear is lame.
Even more because medbay already has a belt that goes inside the bag, it's their unique medkit that can hold surgery tools.
Now Belt Defib on the exoslot would be nice, makes CMO (and everyone else at medbay please) choose between a syringe rifle or a defib for that slot on most cases.

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -205,6 +205,13 @@
 	inhand_icon_state = "medical"
 	worn_icon_state = "medical"
 
+/obj/item/storage/belt/medical/cmo
+	name = "sophisticated medical belt"
+	desc = "A medical belt made with materials used in the inner fabrics of modsuits. It's noticably smaller than regular medical belts."
```
Comment (Fikou) [inline code/game/objects/items/storage/belt.dm:210]:
```suggestion
	desc = "A medical belt made with materials used in the inner fabrics of MODsuits. It's noticably smaller than regular medical belts."
```

**Comment by iwishforducks** [comment]
this is stupid lol can we just make medical belts smaller instead of making the cmo locker a loot piñata

**Comment by Draggeru** [comment]
> this is stupid lol can we just make medical belts smaller instead of making the cmo locker a loot piñata

Someday :Tm:

**Comment by Tastyfish** [comment]
Why not make all job belts smaller? Why do we need to nerf every other department?

**Comment by Draggeru** [comment]
> Why not make all job belts smaller? Why do we need to nerf every other department?

Because all of the belts were just made larger, which is why its a problem.

**Comment by Tastyfish** [comment]
> > Why not make all job belts smaller? Why do we need to nerf every other department?
> 
> Because all of the belts were just made larger, which is why its a problem.

I meant just revert that completely.

**Comment by Draggeru** [comment]
> > > Why not make all job belts smaller? Why do we need to nerf every other department?
> > 
> > 
> > Because all of the belts were just made larger, which is why its a problem.
> 
> I meant just revert that completely.

Lemme know when you get head coder approval for that

**Comment by JohnFulpWillard** [comment]
Unique job equipment is fine, however this is just a straight up upgrade. Having to put your belt down to use a belt item isn't bad, it's already better than having to take the whole bag off.

The main argument I've seen is that Command should be respected more, but giving Command just upgraded equipment doesn't magically make Command be respected, on the contrary I find it encourages people to play these jobs for the equipment, rather than the responsibility, which is not something I like and I doubt players would either.

A good example of unique job equipment is the CMO's biosuit and Hypospray. A biosuit colored in CMO's colors to distinguish them and holds their telebaton, making it fits the CMO more than some MD. The hypospray is 100% unique so can't really be referenced off of anything.
Bad examples of unique job equipment would be the CE's gloves, added with intent to revert a balance change made to insuls by making one specific job exempt from it (like this PR currently is), with it just being a direct upgrade and nothing more, an Engineer could use it and no one would really care for it. It doesn't fit the CE at all, it's just 'upgraded insuls'. They were a little more acceptable back when tablets weren't usable with chunky fingers, but that problem was removed when pda's were merged into tablets.

I think the better way to make Command more respectable is to give them the tools needed to manage departments, not do all the department's work but better, that way players who sign up for it, will sign up for it to manage the department, not to have better tools.

**Comment by Fikou** [comment]
what's the point literally all doctors including the cmo have medkits that can hold all surgery tools

**Comment by SomeoneYouProbablyKnow** [comment]
I think this shows that with belts becoming bulky and storage in general becoming more limited (which IS good) belt items that do not provide storage are gonna be in a bad spot. Instead of focusing on the CMO belt we should instead focus on doing something with those items to either make them really worthwhile or to move them from the belt slot

**Comment by Draggeru** [comment]
> I also agree with GuillaumePrata here- I would prefer the defib change over this.

I've looked into the defib solution and it looks like the compact defib is going to need a lot of code changes, it hard checks for the belt slot.

> what's the point literally all doctors including the cmo have medkits that can hold all surgery tools

That's not a good alternative either, that just fucks medical over for keeping chemical alternatives. Advanced tools arnt a reliable upgrade either because a noticeable amount of the time toxins isn't being done, which is locked behind tech shells.

**Comment by JohnFulpWillard** [comment]
> I've looked into the defib solution and it looks like the compact defib is going to need a lot of code changes, it hard checks for the belt slot.

You add it to the list of things the suit storage can hold- https://github.com/tgstation/tgstation/blob/3e006282e2c511c2c605b2dacf61a5f324e757d5/code/modules/mod/mod_theme.dm#L419 (probably wrong one since CMO gets another)

Since no other maintainer showed support for this PR (or interest in general by commenting) in the two days that it’s been up, and none of my points were even argued against, I’m going to close this, but would nice to see the compact defib change.

**Comment by Mothblocks** [comment]
@JohnFulpWillard For what it's worth I was interested in this PR but was giving it a few days to make sure you saw it

**Comment by Draggeru** [comment]
> > I've looked into the defib solution and it looks like the compact defib is going to need a lot of code changes, it hard checks for the belt slot.
> 
> You add it to the list of things the suit storage can hold-
> 
> https://github.com/tgstation/tgstation/blob/3e006282e2c511c2c605b2dacf61a5f324e757d5/code/modules/mod/mod_theme.dm#L418
> 
> (probably wrong one since CMO gets another)
> Since no other maintainer showed support for this PR (or interest in general by commenting) in the two days that it’s been up, and none of my points were even argued against, I’m going to close this, but would nice to see the compact defib change.

Wrong, adding it to the suit slot dosent work. It hard checks for the belt slot

        else if(slot_flags == ITEM_SLOT_BELT)
            if(user.get_item_by_slot(ITEM_SLOT_BELT) == src)
                ui_action_click()
            else
                to_chat(user, span_warning("Strap the defibrillator's belt on first!"))

**Pull request closed without merging**